### DISCUSSION TITLE
pyMOR Community Meeting 2023-05-19

### DISCUSSION THREAD

**Discussion by HenKlei** [pymor/pymor#2031] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, May 19, at 14:00 CEST and 8:00 EDT, online at:

link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] pyMOR School organization (@pmli)
- [x] issues and PRs for 2023.1 (@pmli)

Moderator: @HenKlei

**Comment by pmli** [comment]
- pyMOR School organization
  - we'll redirect school.pymor.org to 2023.school.pymor.org
  - then advertise it on GitHub discussions
  - fee for coffee breaks and food?
  - sponsor?
  - future pyMOR Schools and/or user meetings (similar to DUNE, FEniCS, NGSolve, etc.)
    - user presentations
    - beginner-friendly presentations
    - hackatons
    - coding challenges
- issues and PRs for 2023.1
  - we should add to the release checklist the assignment of issues

Next meeting: 2023-05-31, 14:00 CEST, 8:00 EDT
Next moderator: @sdrave
